packages/grafana-ui/src/components/Layout/Layout.story.tsx

```diff
@@ -15,31 +15,59 @@ export default {
     docs: {
       page: mdx,
     },
+    controls: {
+      exclude: ['orientation'],
+    },
   },
-};
-
-const justifyVariants = ['flex-start', 'flex-end', 'space-between'];
-
-const spacingVariants = ['xs', 'sm', 'md', 'lg'];
+  args: {
+    justify: 'flex-start',
+    spacing: 'sm',
+    align: 'center',
+    wrap: false,
+    width: '100%',
+  },
+  argTypes: {
+    justify: {
+      control: {
+        type: 'select',
+        options: ['flex-start', 'flex-end', 'space-between', 'center'],
+      },
+    },
+    align: {
+      control: {
+        type: 'select',
+        options: ['flex-start', 'flex-end', 'center', 'normal'],
+      },
+    },
+    spacing: {
+      control: {
+        type: 'select',
+        options: ['xs', 'sm', 'md', 'lg'],
+      },
+    },
+  },
+} as Meta;
 
-export const horizontal = () => {
-  const justify = select('Justify elements', justifyVariants, 'flex-start');
-  const spacing = select('Elements spacing', spacingVariants, 'sm');
+export const Horizontal: Story<LayoutProps> = (args) => {
   return (
-    <HorizontalGroup justify={justify as any} spacing={spacing as any}>
+    <HorizontalGroup {...args}>
       <Button>Save</Button>
       <Button>Cancel</Button>
     </HorizontalGroup>
   );
 };
+Horizontal.args = {
+  orientation: Orientation.Horizontal,
+};
```

Comment:
Sorry for being a complete Horizontal/VerticalGroup newb but is this prop really needed? It wasn't needed before?

Reply:
yeah...you're right. It's not needed at all